app/scripts/modules/core/src/domain/IServerGroup.ts

```diff
@@ -11,9 +12,10 @@ export interface IAsg {
   desiredCapacity: number;
 }
 
-export interface IServerGroup {
+export interface IServerGroup extends IIndexed {
   account: string;
   app?: string;
+  application?: string;
```

Comment:
why both `app` and `application`?

Reply:
Issue resolved by moving to search result specific domain objects.